tests/test_n26_importer.py

```diff
@@ -109,11 +109,14 @@ def test_extract_no_transactions(importer, filename):
 
 
 def assert_transaction(
-    transaction: Transaction,
-    date: datetime,
+    transaction: Transaction,  # type: ignore
+    date: datetime.date,
     payee: str,
     narration: str,
-    postings: List[Tuple[str, Optional[str], Optional[Decimal]]],
+    postings: List[
+        Tuple[str, Optional[str], Optional[Decimal]]
```

Comment:
```suggestion
        Tuple[str, Optional[str | Decimal]]
```

I think this might help with the type annotation error.